web/cobrands/fixmystreet/assets.js

```diff
@@ -1224,9 +1224,10 @@ fixmystreet.message_controller = (function() {
         }
     }
 
-    // Show the reporting form, unless the road responsibility message is visible.
+    /* Show the reporting form, unless the road responsibility message is visible or we are on a
+    Thamemead asset as Thamesmead assets controlling enabling/disabling */
     function enable_report_form() {
-        if ( $('#js-roads-responsibility').is(':visible') ) {
+        if ( $('#js-roads-responsibility').is(':visible') || fixmystreet.bodies.indexOf('Thamesmead') === 0 ) {
```

Comment:
What is calling `enable_report_form` that makes this necessary? Seems odd this wouldn't have come up before somewhere. Is it because Thamesmead code isn't using the standard 'message_controller.asset_not_found' asset code - is there a reason for that?

Reply:
For some reason I had problems at the time with the message_controller, (which may well have been my fault) so I did it the way it is.

Reply:
So on mobile currently if you pick a location that doesn't have an asset, it won't let you continue, but with no explanation as to why. The standard way would get the message shown to the user.